### PR TITLE
Automatically removes the "Download Apps" icon

### PR DESCRIPTION
**Pull request by BE-CH**
### Main update
+ Automatically removes the "Download Apps" icon
+ Added prettier config to match ESLint config
+ Formatted code to match prettier and ESLint

#### Smaller add ons
+ Guild Separator above Download App icon removed
+ When user logs out it will start searching for the Download App icon and the guild separator again to be removed.

### CODE REVIEW

**Comment by BE-CH** [comment]
Requested in comment #6

File: discord-downloadapps-icon.js [hunk 1]
```diff
@@ -0,0 +1,90 @@
+// This is the download app icon
+function findDownloadAppsIconWrapper(stopElement) {
+	let downloadAppsSVG = document.querySelector('[aria-label="Download Apps"]');
```
Comment (yannhodiesne) [inline discord-downloadapps-icon.js:3]: This query selector will break when the display language is not set to English

File: discord-downloadapps-icon.js [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+}
+
+tryRemoveAppIcon();
+
+document.addEventListener('click', (e) => {
+	// when the user clicks log out then start searching again!
+	if (
+		e.target &&
+		e.target.getAttribute('type') === 'submit' &&
+		e.target.innerText.trim() === 'Log Out'
```
Comment (yannhodiesne) [inline discord-downloadapps-icon.js:85]: Same here

File: .prettierrc [hunk 3]
```diff
@@ -0,0 +1,5 @@
+{
+  "singleQuote": true,
+  "tabWidth": 2,
+  "useTabs": true
+}
```
Comment (yannhodiesne) [inline .prettierrc:5]: Thanks for the prettier config!

**Review by yannhodiesne — COMMENTED** [review]
Thank you for your contribution!
You may need to add your 'discord-downloadapps-icon.js' file in the `build.extraResources` key of the package.json file, to bundle it with the release build
I could merge it right away given the quality of your code, but I'd love to see it working when using non-English languages
If you manage to adapt it for other languages, I'll be more than happy to merge it right away!

**Comment by BE-CH** [comment]
> Thank you for your contribution!
> You may need to add your 'discord-downloadapps-icon.js' file in the `build.extraResources` key of the package.json file, to bundle it with the release build
> I could merge it right away given the quality of your code, but I'd love to see it working when using non-English languages
> If you manage to adapt it for other languages, I'll be more than happy to merge it right away!

Yes, I did not think of non-English languages. I will be working on a solution.

**Comment by BE-CH** [comment]
@yannhodiesne I added support for multiple languages. 👍

**Comment by yannhodiesne** [comment]
Nice work, thanks for your contribution!
Can you add `discord-downloadapps-icon.js`to the `extraResources` key inside package.json? I'll merge it and publish a new version once it's done

**Comment by BE-CH** [comment]
> Nice work, thanks for your contribution!
> Can you add `discord-downloadapps-icon.js`to the `extraResources` key inside package.json? I'll merge it and publish a new version once it's done

Yes. Done 👍

**Pull request merged**